frontend/src/translations/en.global.json

```diff
@@ -88,6 +88,64 @@
     }
   },
   "component": {
+    "nodes": {
+      "Validators": {
+        "Validator": "Validator",
```

Comment:
let's define the keys with lower case strings such as `validator`, `fee`, `warning_tip `

Reply:
fied